src/main/java/net/sf/jabref/gui/FileExtensions.java

```diff
@@ -12,36 +14,34 @@
 public enum FileExtensions {
```

Comment:
Slightly unrelated, but didn't @tobiasdiez have some good argumentation that this type of file should be in singular? So `FileExtension`. Not in any way required for the PR though.